pandas/tslib.pyx

```diff
@@ -2416,6 +2416,8 @@ cdef inline parse_timedelta_string(object ts, coerce=False):
     elif have_dot:
 
         if (len(number) or len(frac)) and not len(unit) and current_unit is None:
+            if coerce:
+                return iNaT
```

Comment:
See https://github.com/pydata/pandas/pull/10265/files#diff-933180870dd152b775952fc54a99dba7R5928

for the test that was failing.